package.json

```diff
@@ -15,6 +15,7 @@
     "geoserver-node-client": "0.0.5",
     "node-fetch": "^2.6.1",
     "nodemailer": "^6.6.2",
+    "pg": "^8.7.3",
```

Comment:
please rebase and make sure all the old dependencies will dissappear here. You will now need to move the package.json and the lockfile to your new specific worker directory.

Reply:
alright. Done